### ISSUE TITLE
[Question] How to set prefs for a firefox playwright?

### ISSUE THREAD

**Issue by Somberor** [microsoft/playwright#6306]
I would like to set costume prefs before starting the firefox instance,
prefs such as `user_pref("general.oscpu.override", "Win32");`
How can I do that?

**Comment by yury-s** [comment]
You can use [`firefoxUserPrefs`](https://playwright.dev/docs/api/class-browsertype?_highlight=launch#browsertypelaunchoptions) for that:

```js
await browserType.launch({firefoxUserPrefs: {"general.oscpu.override", "Win32"}});
```

**Comment by nlile** [comment]
@yury-s is there a reason why `firefoxUserPrefs` is not a parameter for `launchPersistentContext`? Did some searches and can't seem to find anyone complaining about it.

**Comment by mxschmitt** [comment]
Sounds like a valid feature request, please open a new issue for it, thanks!

**Comment by nlile** [comment]
Sure thing, just opened #7687
